src/mixins/stylable.js

```diff
@@ -0,0 +1,12 @@
+import StylePropable from './style-propable';
+
+export default function(target) {
+  target.propTypes = {
+    ...StylePropable.propTypes,
+    ...target.propTypes,
+  };
+
+  target.prototype.mergeStyles = StylePropable.mergeStyles;
+  target.prototype.mergeAndPrefix = StylePropable.mergeAndPrefix;
```

Comment:
My goal was to just expose the functionality of the mixin 1-1, I didn't really look too much into it.

Also, if it's of any interest, the code-decorators library I have in the dependencies does have a deprecation decorator, which might be useful.

Also, should I make another folder for the decorators? `src/decorators/`?